### DISCUSSION TITLE
Alternative icon for Chromium, based on devtools org logo

### DISCUSSION THREAD

**Discussion by ZuBB** [elrumo/macOS_Big_Sur_icons_replacements#1476] [Icon request] [answered]
we already have an icon that is "drawing-based," but the logo from ChromeDevTools has even more engineering spirit. Maybe someone is willing to spare a few minutes to convert `.svg` to `.icns`  and add padding(this is the main issue) to it.

<img width="1064" alt="image" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/597016/98f69ac9-76a1-46cd-9e66-88ba80dc40e7">

https://github.com/ChromeDevTools/devtools-logo/tree/master

**Comment by vikasbhatia** [comment, marked as answer]
Just made these. You can download the ICNS from [my profile](https://macosicons.com/#/u/sidewinder) once it's approved, but here's the PNG until then:

<img width="512" alt="Chromium-2" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/d698dc14-92cc-42c8-8b88-aa1c7ef73242">
<img width="512" alt="Chromium" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/9ca0c9a9-6486-4121-a98f-b80fdf8d148c">

**Reply by ZuBB** [reply]
this looks amazing @vikasbhatia. thanks a lot
